compiler/noirc_driver/src/lib.rs

```diff
@@ -70,6 +70,11 @@ pub struct CompileOptions {
     #[arg(long, hide = true)]
     pub show_ssa: bool,
 
+    /// If `show_ssa` is true, setting this will only show SSA passes whose name
+    /// contains the provided string.
```

Comment:
It will also print names if `show_ssa` is false, _but_ `options.show_ssa_pass_name` is given, won't it?

Reply:
Maybe like this?

> Only show SSA passes whose name contains the provided string. If `show_ssa` is true, this setting takes precedence.

Reply:
> If show_ssa is true

Did you mean "if show_ssa is false"?

Reply:
I used your comment but changed a bit. Let me know if it's okay.